tests/hypothesis/test_concat.py

```diff
@@ -44,27 +44,29 @@ def test_concat(  # pragma: no cover
 
     if how == "horizontal":
         df_pl = (
-            nw.LazyFrame(df_polars)
+            nw.from_native(df_polars)
+            .lazy()
```

Comment:
I kept this like it was #331 for now, but I'm not 100% sure why we are testing here (and other tests below) the `LazyFrame` interface, when both outputs are not lazy.

What would we miss it we just had `from_native()`?

Reply:
😄 this looks like a mistake tbh, I think `from_native` should be fine, thanks for noticiing

Reply:
I _might_ have been chasing coverage with some of these...but this `.lazy().collect()` definitely feels useless

Reply:
> I might have been chasing coverage with some of these

I'll try to delete some of them and see if coverage is happy 😅